tests/src/system/basic/WskActionTests.scala

```diff
@@ -244,20 +244,20 @@ class WskActionTests
             val name = "ping"
             val samplePackage = "samples"
             val sampleAction = "helloWorld"
-            val fullActioName = s"/$guestNamespace/$samplePackage/$sampleAction"
+            val fullQualifiedName = s"/$guestNamespace/$samplePackage/$sampleAction"
```

Comment:
I think I noted this in another comment - if not, here goes again:
I don't think we need to create any assets here.
The test should fail with a ERROR exit code (1) so if the CLI permits the operation, it would otherwise fail with 148.